src/Storages/System/StorageSystemDictionaries.cpp

```diff
@@ -52,6 +51,14 @@ catch (const DB::Exception &)
 
 }
 
+StorageSystemDictionaries::StorageSystemDictionaries(const StorageID & storage_id_)
+    : DB::IStorageSystemOneBlock<StorageSystemDictionaries>(storage_id_)
+{
+    VirtualColumnsDescription virtuals;
+    virtuals.addEphemeral("key", std::make_shared<DataTypeString>(), "");
```

Comment:
Not related to the diff, but seems that the name "key" violates the naming convention for virtual columns?

Reply:
Yes, but let's not introduce backward incompatibility.